packages/elements-core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stoplight/elements-core",
-  "version": "7.2.5",
+  "version": "7.2.6",
```

Comment:
Please consider creating separate PR for a release. I think that Mateusz pushed a change in the meantime.

Reply:
I think we can do it for a quick fixes like this. The branch is up to date and only a single (internal) package is changed, so it seems like a ok thing to do 🤷‍♂️